internal/powerbi/resource_refresh_schedule_test.go

```diff
@@ -129,11 +132,11 @@ func TestAccRefreshSchedule_basic(t *testing.T) {
 				),
 			},
 			// final step checks importing the current state we reached in the step above
-			{
-				ResourceName:      "powerbi_refresh_schedule.test",
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
+			// {
+			// 	ResourceName:      "powerbi_refresh_schedule.test",
+			// 	ImportState:       true,
+			// 	ImportStateVerify: false,
```

Comment:
Commented out test

This is also highlights a concern with adding another required identifier field, im not sure what the impacts on importing become